subprojects/docs/src/docs/userguide/dep-man/02-declaring-dependency-versions/single_versions.adoc

```diff
@@ -0,0 +1,73 @@
+[[single-version-declarations]]
+= Single version declarations
+
+The simplest version declaration is a _simple string_ representing the version to use.
+Gradle supports different ways of declaring a version string:
+
+* An exact version: e.g. `1.3`, `1.3.0-beta3`, `1.0-20150201.131010-1`
+* A Maven-style version range: e.g. `[1.0,)`, `[1.1, 2.0)`, `(1.2, 1.5]`
+** The `[` and `]` symbols indicate an inclusive bound; `(` and `)` indicate an exclusive bound.
+** When the upper or lower bound is missing, the range has no upper or lower bound.
+** The symbol `]` can be used instead of `(` for an exclusive lower bound, and `[` instead of `)` for exclusive upper bound. e.g `]1.0, 2.0[`
+* A _prefix_ version range: e.g. `1.+`, `1.3.+`
+** Only versions exactly matching the portion before the `+` are included.
+** The range `+` on it's own will include any version.
+* A `latest-status` version: e.g. `latest.integration`, `latest.release`
+** Will match the highest versioned module with the specified status. See link:{javadocPath}/org/gradle/api/artifacts/ComponentMetadata.html#getStatus--[ComponentMetadata.getStatus()].
+* A Maven `SNAPSHOT` version identifier: e.g. `1.0-SNAPSHOT`, `1.4.9-beta1-SNAPSHOT`
```

Comment:
I think the new strict version notation "!!" should also be in this list, wdyt?

Reply:
mmm, I'm not sure. This notation is described below. The idea was to describe:

- the simplest declaration
- then semantics of ordering
- then semantics of strict (and the short-hand notation)